runtime/doc/filetype.txt

```diff
@@ -512,37 +512,44 @@ Local mappings:
 
 MAN					*ft-man-plugin* *:Man* *man.vim*
 
-Displays a manual page in a nice way.  Also see the user manual
-|find-manpage|.
+View manpages from the comfort of vim. Features include syntax highlighting,
+smart autocompletion, multilingual support, and manpage jumping.
+Also see |find-manpage|.
 
-To start using the ":Man" command before any manual page was loaded, source
-this script from your startup vimrc file: >
+You can use neovim as a manpager with >
 
-	runtime ftplugin/man.vim
-
-Options:
-'iskeyword'	the '.' character is added to be able to use CTRL-] on the
-		manual page name.
+     export MANPAGER="nvim -c 'set ft=man' -"
 
 Commands:
-Man {name}	Display the manual page for {name} in a window.
-Man {number} {name}
-		Display the manual page for {name} in a section {number}.
-
-Global mapping:
-<Leader>K	Displays the manual page for the word under the cursor.
+Man {name}		Display the manpage for {name} in a new tab.
+Man {sect} {name}	Same as above except specify the section.
+Man {name}({sect})	Alternate syntax to above. Exists so that you can see
+			the section of the manpage you are completing.
+Man {path}		Open the manpage specified by path, if it's in the current
+			directory, specify the leading ./
+Global Mappings:
+<Plug>(Man)             Jump to the manpage for the <cWORD> under the
+                        cursor in a new tab. Takes a count as the manpage
+			section.
 
 Local mappings:
-CTRL-]		Jump to the manual page for the word under the cursor.
-CTRL-T		Jump back to the previous manual page.
-q		Same as ":quit"
+CTRL-]			Jump to the manpage for the <cWORD> under the cursor. Takes a count
+                        as the manpage section.
+K			Same as CTRL-].
+CTRL-T			Jump back to the previous manpage.
+q			Close the window.
 
-To enable folding use this: >
-  	let g:ft_man_folding_enable = 1
-If you do not like the default folding, use an autocommand to add your desired
-folding style instead.  For example: >
-        autocmd FileType man setlocal foldmethod=indent foldenable
+Variables:
+g:no_man_maps
+			If set, no mappings are created in man buffers.
+
+g:ft_man_folding_enable
+                        If set, manpages are folded with foldmethod=indent and
+                        foldnestmax=1.
 
+If you do not like the default folding, use an autocommand to add your desired
```

Comment:
Is this really necessary? I do not think anyone who uses vim (or even got that far into the docs) needs to know how to use `:autocmd` to set their favourite folding style.